extension/src/commands/CreateProjectCommand.ts

```diff
@@ -47,7 +47,7 @@ export class CreateProjectCommand extends Command {
             const success = await this.runSteps(metadata);
             if (success) {
                 await this.createProject(metadata);
-                const openInNewWindow = !(folders && folders.length);
+                const openInNewWindow = folders && folders.length;
```

Comment:
`workspaceFolders` has below options:
- undefined: no workspace opened.
- empty array: current workspace containing no root folders.
- non-empty array: single-root workspace or multi-root workspace.

Do you want to have different behavior for undefined and empty array?
- []

Reply:
How about opening the newly created project in the current window only when there is no workspace open? for empty array and non-empty array, we'd better to open a new window since the current workspace contains files.